File: Coder Revolt - Web Browser Open Source/Coder Revolt - Web Browser Open Source/exit page.vb
Public Class exit_page
    Private Sub Button1_Click(sender As System.Object, e As System.EventArgs) Handles Button1.Click
        Application.Exit()

    End Sub

    Private Sub Button2_Click(sender As System.Object, e As System.EventArgs) Handles Button2.Click
        Me.Close()
        Main.Enabled = True
        Main.Show()


    End Sub
End Class

File: C# Converted Code V.1/exit page.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
public class exit_page
{
	private void Button1_Click(System.Object sender, System.EventArgs e)
	{
		Application.Exit();

	}

	private void Button2_Click(System.Object sender, System.EventArgs e)
	{
		this.Close();
		Main.Enabled = true;
		Main.Show();


	}


	private void exit_page_Load(object sender, EventArgs e)
	{
	}
	public exit_page()
	{
		Load += exit_page_Load;
	}
}